sycl/test/esimd/intrins_trans.cpp

```diff
@@ -196,7 +197,7 @@ SYCL_EXTERNAL vec<unsigned short, 8> get8ui16() SYCL_ESIMD_FUNCTION;
 SYCL_EXTERNAL vec<unsigned char, 8> get8ui8() SYCL_ESIMD_FUNCTION;
 
 SYCL_EXTERNAL void
-test_mem_intrins(uint64_t addr, const vec<float, 8> &xf,
```

Comment:
why the change here? are we sure we nothing regressed?

Reply:
'addr' was used only to test __esimd_svm_block_ld/st and __esimd_svm_block_ld_unaligned. Those intrinsics accepted uint64_t.
This PR replaced/changed those intrinsics with intrinsics accepting pointers. Thus this change was needed.

Reply:
got it, thx